stacks/api/passport/apikey/interface.ts

```diff
@@ -0,0 +1,10 @@
+import {ObjectId} from "@spica-server/database";
+
+export interface ApiKey {
+  _id?: ObjectId;
+  key?: string;
+  name: string;
+  description?: string;
+  policies: Array<string>;
```

Comment:
What do you mean by ValueObjects?

Reply:
https://en.wikipedia.org/wiki/Value_object

Value Objects allows us extendibility for possible future improvements.

Reply:
That would make things hard since we will be storing them in the database by duplicating. Passing things as primitives or plain objects much more maintainable and readable in javascript. We separate data and behavior in most cases.